Go Releaser Procedure.md

```diff
@@ -0,0 +1,184 @@
+
+### 1. Init path
+
+- Locate in the your project development git folder
+
+### 2. Checkout and prepare adequate branch
+
+- The release branch has to be created as defined in SDLC procedure; taking into account to be up to date from develop branch.
+
+  Then, and in terms of the current release stage, the adequate branch should be created and published.
+
+  So, checkout adequate branch for current release and update with latest changes.
+
+### 3. Set adequate "VERSION" in code file
+
+- Set the adequate release version in code file
+
+  Update in "utils/version.go" the adequate version: `const VERSION = [CURRENT_VERSION]`
+
+  Samples:
+
+  ```go
+  const VERSION = "0.6.0-alpha"
+  const VERSION = "0.6.0-alpha.1"
+  const VERSION = "0.6.0-beta"
+  const VERSION = "0.6.0-beta.1"
+  const VERSION = "0.6.0-rc"
+  const VERSION = "0.6.0-rc1"
+  const VERSION = "0.6.0"
+  ```
+
+- Add/Commit version file
+
+  ```bash
+  git add "utils/version.go"
+  git commit -m "Update version to [CURRENT_VERSION]"
+  ```
+
+- Publish the release branch, depending of current case:
+
+  Samples:
+
+  ```bash
+  git push release/0.6.0-alpha
+  git push release/0.6.0-alpha.1
+  git push release/0.6.0-beta
+  git push release/0.6.0-beta.1
+  git push release/0.6.0-rc
+  git push release/0.6.0-rc1
+  git push release/0.6.0
+  ```
+
+### 4. ONLY IN RTM CASE. Prepare master branch
+
+In order to create the latest release and to public as production ready:
+
+- Create Pull Request into MASTER and wait for approval prior to merge and continue the procedure
+
+Once approved and merged:
+
+- Checkout the master branch
+
+  ```bash
+  git checkout master
+  ```
+
+- Update with the latest changes
+
+  ```bash
+  git pull -r
+  ```
+
+### 5. Tag the release
+
+- Tag the new release
+
+  ```bash
+  git tag -a v[CURRENT_VERSION] -m "Concerto v[CURRENT_VERSION]"
+  ```
+
+  Samples:
+
+  ```bash  
+  git tag -a v0.6.0-alpha -m "Concerto v0.6.0-alpha"
+  git tag -a v0.6.0-alpha.1 -m "Concerto v0.6.0-alpha.1"
+  git tag -a v0.6.0-beta -m "Concerto v0.6.0-beta"
+  git tag -a v0.6.0-beta.1 -m "Concerto v0.6.0-beta.1"
+  git tag -a v0.6.0-rc -m "Concerto v0.6.0-rc"
+  git tag -a v0.6.0-rc1 -m "Concerto v0.6.0-rc1"
+  git tag -a v0.6.0 -m "Concerto v0.6.0"
+  ```
+
+- Publish tag
+
+  ```bash
+  git push origin v[CURRENT_VERSION]
+  ```
+
+  Samples:
+
+  ```bash
+    git push origin v0.6.0-alpha
+    git push origin v0.6.0-alpha.1
+    git push origin v0.6.0-beta
+    git push origin v0.6.0-beta.1
+    git push origin v0.6.0-rc
+    git push origin v0.6.0-rc1
+    git push origin v0.6.0
+  ```
+
+### 6. ONLY IN RTM CASE. Merge into develop branch
+
+In addition, and once the final stable release has been merged into master branch as well as tagged; the master branch must be merged into develop and this should be done through a PR.
```

Comment:
We usually do this after publishing the release (which we do here with goreleaser), and though the order does not matter, it makes sense to document it at the end of the procedure so as not to 'break its flow'